.eslintrc.js

```diff
@@ -89,7 +89,10 @@ module.exports = {
       files: ['src/mobile/**/*.js', 'src/mobile/**/*.jsx'],
       extends: ['plugin:react/recommended', 'plugin:react-hooks/recommended'],
       plugins: ['react-native'],
-      rules: { 'no-use-before-define': 'off' },
+      rules: {
+        'no-use-before-define': 'off',
+        'react/jsx-filename-extension': [1, { allow: 'as-needed' }],
```

Comment:
Why are we making this change?  Why not always use `.jsx` for components?

Reply:
@humphd I just check it out and you are right there is no need to do that `.jsx` work fine